lychee-lib/src/types/input.rs

```diff
@@ -133,13 +141,25 @@ impl Input {
                     ignore_case: glob_ignore_case,
                 }
             } else {
-                InputSource::FsPath(value.into())
+                let path = PathBuf::from(value);
+                if path.exists() {
+                    InputSource::FsPath(value.into())
```

Comment:
instead of value.into(), can you use path directly?

Reply:
Thanks, changed.